code/_globalvars/sticker.dm

```diff
@@ -0,0 +1,21 @@
+///Global list of stickers by series
+GLOBAL_LIST(stickers_by_series)
+
+///Fill globals
+/proc/fill_sticker_globals()
+	if(length(GLOB.stickers_by_series))
+		return
+	/*
+		Build sticker GLOB.stickers_by_series
+		just index each series flag with a list of associated sticker objects
+	*/
+	var/list/temp = list()
+	var/series = STICKER_SERIES_1 //Make sure you update this if you add more series
+	for(var/obj/item/sticker/S as() in subtypesof(/obj/item/sticker))
```

Comment:
avoid using this abitrary name. I recommend using `each_sticker` rather than `S`
also, use `as anything` than `as()`